staging/src/k8s.io/apimachinery/pkg/util/naming/from_stack_test.go

```diff
@@ -31,12 +31,12 @@ func TestGetNameFromCallsite(t *testing.T) {
 		{
 			name:            "ignore-package",
 			ignoredPackages: []string{"k8s.io/apimachinery/pkg/util/naming"},
-			expected:        "testing/testing.go:827",
+			expected:        "testing/testing.go:865",
 		},
 		{
 			name:            "ignore-file",
 			ignoredPackages: []string{"k8s.io/apimachinery/pkg/util/naming/from_stack_test.go"},
-			expected:        "testing/testing.go:827",
+			expected:        "testing/testing.go:865",
```

Comment:
this is a wacky test

Reply:
simplified in https://github.com/kubernetes/kubernetes/pull/74735 to not need rework on go version bumps

Reply:
removed the change from here, as #74735 is way better.